.circleci/config.yml

```diff
@@ -114,7 +114,7 @@ workflows:
                 name: "Public Beta Release"
                 filters:
                     branches:
-                        only: Release/*
+                        only: /Release\/.*/
```

Comment:
@neelmistry94 can you update this PR to explain in more detail how this change works? What was broken about the old way? How does the new way fix it?

Reply:
@neelmistry94 I still don't understand what was broken. Can you give an example of a branch name that didn't match the original pattern and explain why it didn't match and then explain why it does match the new pattern?

Reply:
So what I had previously without the "//" it was not recognizing it as a regex. 

So when we push up a branch called Release/v10.0.0.... it did not work because it was not a valid regex. The addition of the slashes will make it correct